EIPS/eip-1193.md

```diff
@@ -7,6 +7,7 @@ status: Draft
 type: Standards Track
 category: Interface
 created: 2018-06-30
+requires: 155, 695, 1102, 1474
```

Comment:
@danfinlay even if not changing the chainid, adding 1102 and 1474 as a requirement here may make sense.